kimchi/src/prover_index.rs

```diff
@@ -211,7 +211,9 @@ pub mod testing {
             override_srs_size,
             |d1: D<G::ScalarField>, size: usize| {
                 let log2_size = size.ilog2();
+                // Run test_srs_serialization test to generate test SRS & enable this
                 let mut srs = if log2_size <= precomputed_srs::SERIALIZED_SRS_SIZE {
+                    // @volhovm: on my machine for d=16 (2.2Mb SRS file) this takes 110s. Why???
```

Comment:
As discussed, it is because of the missing `--release` flag.

Reply:
We should document this somewhere.

Reply:
@mrmr1993 @dannywillems Adding the following into the top-lvl `Cargo.toml` fixes the issue:
```
[profile.test]
opt-level = 1 # Otherwise SRS deserialization in tests takes forever
```
and I don't think it's terribly unreasonable, unless we really want our non-release test build to be fully unoptimised?

Reply:
> We should document this somewhere.

https://github.com/o1-labs/proof-systems/blob/develop/CONTRIBUTING.md#development

Reply:
I put a comment with a note and the link to the `get_srs` function.